### ISSUE TITLE
Links are not clickable

### ISSUE THREAD

**Issue by simonlindblad** [holmgr/chapeau#3]
When using the links with the syntax `[name](url)` the links are not clickable in the PDF. It would be a nice feature to have. :)

**Comment by holmgr** [comment]
Yes it really would. However there is quite a few problems in this area when it comes to the tools for generating pdfs from html. Currently PhantomJs is used for the conversion but I have also tried wkhtmltopdf and that also does not create working links. So as of now it does not seem to be fixable :(

**Comment by simonlindblad** [comment]
Okey, too bad!
